tools/resources/resources.json

```diff
@@ -128,7 +128,12 @@
         "webapi/webapi-vehicleinfo-ivi-tests/resources",
         "webapi/webapi-webcl-nonw3c-tests/resources",
         "webapi/webapi-webrtc-w3c-tests/resources",
-        "webapi/webapi-webspeech-w3c-tests/resources"
+        "webapi/webapi-webspeech-w3c-tests/resources",
```

Comment:
No, I don't want manually copy them on demand.

If resources.json is too long, you can split them, e.g. `testharness.json`, `webrunner.json`, etc.